common/cmake/devices.cmake

```diff
@@ -993,16 +999,16 @@ function(ADD_FPGA_TARGET)
   endforeach()
 
   if(NOT ${ADD_FPGA_TARGET_NO_SYNTHESIS})
-    set(COMPLETE_YOSYS_SYNTH_SCRIPT "tcl ${YOSYS_SYNTH_SCRIPT}")
+    set(COMPLETE_YOSYS_SYNTH_SCRIPT "plugin -i xdc; plugin -i fasm; tcl ${YOSYS_SYNTH_SCRIPT}")
```

Comment:
Can these plugin calls be made from inside the `YOSYS_SYNTH_SCRIPT`?  If yes, please do it there.